script.js

```diff
@@ -1,4 +1,81 @@
+// Create game states
+var playerToRollDice = "rollDice";
+var playerChooseDiceOrder = "chooseDiceOrder";
+
+var player2ToRollDice = "rollDice2";
+var player2ChooseDiceOrder = "chooseDiceOrder2";
+
+var compareScores = "compareScores";
+
+// Default game state
+var currentGameMode = playerToRollDice;
+
+var player1DiceRoll1, player1DiceRoll2;
+var player2DiceRoll1, player2DiceRoll2;
+
+var player1DiceOrder, player2DiceOrder;
```

Comment:
while this method of declaring multiple variables work, I'd much prefer if you would just have them line by line as opposed to them being in one line. It's generally more readable that way.

```
var player1DiceOrder
var player2DiceOrder
```